saleor/graphql/product/mutations/products.py

```diff
@@ -556,6 +555,7 @@ class ProductCreateInput(ProductInput):
         ),
         required=False,
     )
+    base_price = Decimal(description="Default price for product variant.",)
```

Comment:
What about the `productUpdate` mutation? It uses `ProductInput` and for now we want to use it to update prices of simple products (with the one default variant).

Reply:
I just restored old logic.

Reply:
So to not break creating products in the dashboard, we should have the `base_price` field in both mutations - for creating and updating products. Now we have it only in mutation to create. If I'm correct OFC, it's Friday afternoon ;)